src/kbmod/masking.py

```diff
@@ -2,6 +2,44 @@
 """
 
 import kbmod.search as kb
+from kbmod.search import RawImage
+
+
+def mask_trajectory(trj, stack, r):
+    """Will apply a circular mask of radius `r` around
+    the trajectory's predicted position in each image.
+
+    Attributes
+    ----------
+    trj : `Trajectory`
+        The trajectory along which to apply the mask.
+    stack : `ImageStack`
+        The stack of images to apply the mask to.
+    r : `int`
+        The radius of the circular mask to apply.
+
+    Returns
+    -------
+    stack : `ImageStack`
+        The stack after the masks have been applied.
+    """
+
+    width = stack.get_width()
+    height = stack.get_height()
+
+    for i in range(stack.img_count()):
+        time = stack.get_single_image(i).get_obstime() - stack.get_single_image(0).get_obstime()
+        origin_of_mask = (int(trj.get_x_pos(time) + 0.5), int(trj.get_y_pos(time) + 0.5))
+        img = stack.get_single_image(i)
```

Comment:
As a very small point. You could switch the order to make things slightly shorter and more efficient by:
- Moving `img = stack.get_single_image(i)` to the front of the loop.
- Computing time as `time = img.get_obstime() - stack.get_single_image(0).get_obstime()`

Reply:
Changed. Where do we get efficiency from with the new approach?

Reply:
The efficiency would be in getting the reference to the image once instead of twice. To be honest, I don't expect it to really matter at all.